TensorKitSectors/src/trivial.jl

```diff
@@ -0,0 +1,36 @@
+"""
+    Trivial
+
+Singleton type to represent the trivial sector, i.e. the trivial representation of the
+trivial group. This is equivalent to `Rep[ℤ₁]`, or the unit object of the category `Vect` of
+ordinary vector spaces.
+"""
+struct Trivial <: Sector end
+
+Base.show(io::IO, ::Trivial) = print(io, "Trivial()")
```

Comment:
Is this method actually necessary? It seems like default Julia printing would cover this.